File: jscomp/test/test_global_print.js
'use strict';


var List = do
  u: 3
end;

var X = do
  List: List
end;

var Hashtbl = --[ alias ]--0;

var V = --[ alias ]--0;

exports.X = X;
exports.Hashtbl = Hashtbl;
exports.V = V;
--[ No side effect ]--

File: jscomp/test/test_global_print.lua
__console = {log = print};


List = {
  u = 3
};

X = {
  List = List
};

Hashtbl = --[[ alias ]]0;

V = --[[ alias ]]0;

exports = {};
exports.X = X;
exports.Hashtbl = Hashtbl;
exports.V = V;
return exports;
--[[ No side effect ]]
